### DISCUSSION TITLE
Elafgifter mangler i beregningerne

### DISCUSSION THREAD

**Discussion by Birchedahl** [rndfm/elspotpris#173] [General]
Elafgifter er pludselig udeladt fra alle beregninger - De bør indsættes igen.

**Comment by rndfm** [comment]
Du har ikke bare slået elafgift fra?

Husk at elafgiften er ret lav pt.

**Reply by Birchedahl** [reply]
![image](https://github.com/rndfm/elspotpris/assets/136216647/a4dfb1e1-457b-4e03-be95-d2b23145c8a3)

**Comment by Birchedahl** [comment]
I  plejer at have den med

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 11. juni 2023 17:28
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Du har ikke bare slået elafgift fra?

Husk at elafgiften er ret lav pt.

**Comment by rndfm** [comment]
Og det har vi selvfølgelig også stadig

**Comment by Birchedahl** [comment]
Nej – elafgiften er udeladt totalt alle vegne. Og om 14 dage stiger den igen.

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 11. juni 2023 20:35
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Og det har vi selvfølgelig også stadig

**Comment by rndfm** [comment]
https://github.com/rndfm/elspotpris/assets/46886958/edae44e4-ea96-405b-bfac-4af965884179

**Comment by rndfm** [comment]
![image](https://github.com/rndfm/elspotpris/assets/46886958/dc642b71-d60b-4c28-8b04-c0e99d4e2bf7)

Eneste grund til at elafgiften skulle mangle er hvis man fjerner den på siden via afkrydsningsfeltet.

Kan du evt. tage nogle skærmbilleder af det du ser?

**Comment by Birchedahl** [comment]
Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 10:17
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

 <https://user-images.githubusercontent.com/46886958/245075037-dc642b71-d60b-4c28-8b04-c0e99d4e2bf7.png> 

Eneste grund til at elafgiften skulle mangle er hvis man fjerner den på siden via afkrydsningsfeltet.

Kan du evt. tage nogle skærmbilleder af det du ser?

**Reply by rndfm** [reply]
Jeg tror muligvis du har svaret emailen med et billede? Det får vi bare ikke rigtigt med.

Hvis man skal indsætte billeder skal man svare i tråden på github direkte.

**Comment by Birchedahl** [comment]
Billedet viser at der er sat hak ved elafgift og at der ikke i beregningen vises nogen el-afgift.

 

Altså – Jeg kan ikke gøre det anderledes og el-afgiften mangler på ALLE el-selskaber.

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 14:33
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Jeg tror muligvis du har svaret emailen med et billede? Det får vi bare ikke rigtigt med.

Hvis man skal indsætte billeder skal man svare i tråden på github direkte.

**Comment by rndfm** [comment]
Det lyder spøjst? Er det app versionen du bruger?

Kan måske være fordi det er en ældre version.

Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version.

**Comment by Birchedahl** [comment]
Nej Det er direkte på nettet – elspotpris.dk

 

OG i starten havde I el-afgiften med. Hvornår den er forsvundet fra beregningerne, kan jeg ikke sige – men den er bare ikke med længere.

 

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:08
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Det lyder spøjst? Er det app versionen du bruger?

Kan måske være fordi det er en ældre version.

Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version.

**Comment by Birchedahl** [comment]
Jeg har fundet årsagen.

 

Det er browseren. Web-siden vises ikke korrekt i Google Chrome mere.

Jeg har altid brugt Chrome, men det er ud til, at jeg nu skal skifte til Edge browseren.

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:08
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Det lyder spøjst? Er det app versionen du bruger?

Kan måske være fordi det er en ældre version.

Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version.

**Reply by rndfm** [reply]
> Jeg har fundet årsagen. Det er browseren. Web-siden vises ikke korrekt i Google Chrome mere. Jeg har altid brugt Chrome, men det er ud til, at jeg nu skal skifte til Edge browseren. Med venlig hilsen Anne Birchedahl Elfengård Åtoften 28 2680 Solrød str. 2390 0028 Fra: Christian Reinholdt ***@***.***> Sendt: 12. juni 2023 17:08 Til: rndfm/elspotpris ***@***.***> Cc: Birchedahl ***@***.***>; Author ***@***.***> Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173) Det lyder spøjst? Er det app versionen du bruger? Kan måske være fordi det er en ældre version. Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version. — Reply to this email directly, view it on GitHub <[#173 (comment)](https://github.com/rndfm/elspotpris/discussions/173#discussioncomment-6153548)> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/BAPIARYMSC63R7OA746C4ELXK4WE7ANCNFSM6AAAAAAZCCWNSI> . You are receiving this because you authored the thread. <https://github.com/notifications/beacon/BAPIAR3Y6OUFPOWWN4W6GVDXK4WE7A5CNFSM6AAAAAAZCCWNSKWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQALXSUY.gif> Message ID: ***@***.*** ***@***.***> >

Har du prøvet ctrl + F5 (eller refresh knappen i browseren?)

**Comment by Birchedahl** [comment]
Selvfølgelig. Og uden resultat

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:22
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Jeg har fundet årsagen. Det er browseren. Web-siden vises ikke korrekt i Google Chrome mere. Jeg har altid brugt Chrome, men det er ud til, at jeg nu skal skifte til Edge browseren. Med venlig hilsen Anne Birchedahl Elfengård Åtoften 28 2680 Solrød str. 2390 0028 Fra: Christian Reinholdt @.> Sendt: 12. juni 2023 17:08 Til: rndfm/elspotpris @.> Cc: Birchedahl @.>; Author @.> Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173 <https://github.com/rndfm/elspotpris/discussions/173> ) Det lyder spøjst? Er det app versionen du bruger? Kan måske være fordi det er en ældre version. Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version. — Reply to this email directly, view it on GitHub <#173 (comment) <https://github.com/rndfm/elspotpris/discussions/173#discussioncomment-6153548> > , or unsubscribe https://github.com/notifications/unsubscribe-auth/BAPIARYMSC63R7OA746C4ELXK4WE7ANCNFSM6AAAAAAZCCWNSI . You are receiving this because you authored the thread. https://github.com/notifications/beacon/BAPIAR3Y6OUFPOWWN4W6GVDXK4WE7A5CNFSM6AAAAAAZCCWNSKWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQALXSUY.gif Message ID: @.*** @.***> >

Har du prøvet ctrl + F5 (eller refresh knappen i browseren?)

**Comment by Birchedahl** [comment]
Jeg kører den bare i den anden browser fremover

Jeg har fundet årsagen. Det er browseren. Web-siden vises ikke korrekt i Google Chrome mere. Jeg har altid brugt Chrome, men det er ud til, at jeg nu skal skifte til Edge browseren. Med venlig hilsen Anne Birchedahl Elfengård Åtoften 28 2680 Solrød str. 2390 0028 Fra: Christian Reinholdt @.> Sendt: 12. juni 2023 17:08 Til: rndfm/elspotpris @.> Cc: Birchedahl @.>; Author @.> Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173 <https://github.com/rndfm/elspotpris/discussions/173> ) Det lyder spøjst? Er det app versionen du bruger? Kan måske være fordi det er en ældre version. Prøv evt. at genopfriske med ctrl + F5 på PC eller træk ned fra toppen af siden på mobil for at hente en ny version. — Reply to this email directly, view it on GitHub <#173 (comment) <https://github.com/rndfm/elspotpris/discussions/173#discussioncomment-6153548> > , or unsubscribe https://github.com/notifications/unsubscribe-auth/BAPIARYMSC63R7OA746C4ELXK4WE7ANCNFSM6AAAAAAZCCWNSI . You are receiving this because you authored the thread. https://github.com/notifications/beacon/BAPIAR3Y6OUFPOWWN4W6GVDXK4WE7A5CNFSM6AAAAAAZCCWNSKWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQALXSUY.gif Message ID: @.*** @.***> >

Har du prøvet ctrl + F5 (eller refresh knappen i browseren?)

**Reply by rndfm** [reply]
Jeg bruger selv Chrome browseren og har ikke problemet. Det er da lidt mystisk.

**Comment by Birchedahl** [comment]
Ja, det er det faktisk. Det retter sig måske med næste opdatering.

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:28
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Jeg bruger selv Chrome browseren og har ikke problemet. Det er da lidt mystisk.

**Comment by Birchedahl** [comment]
Prøv lige at teste om du også får nogle andre nettariffer, hvis du skifter til Edgebrowseren

 

Det gør jeg nemlig.

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:28
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Jeg bruger selv Chrome browseren og har ikke problemet. Det er da lidt mystisk.

**Comment by rndfm** [comment]
Det er helt ens for mig i edge og chrome. (Edge er en chrome browser indenunder)

**Comment by Birchedahl** [comment]
Det er det ikke for mig – I edge er nettarif C lige nu 0,73588 mens Chrome viser 0,69350

 

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 17:43
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Det er helt ens for mig i edge og chrome. (Edge er en chrome browser indenunder)

**Comment by rndfm** [comment]
Begge er valgt med samme netselskab?

*Med venlig hilsen | best regards*
*Christian Reinholdt*
Web-professionel og autodidakt alt-mulig-mand.

tlf. 31718115
***@***.***

**Comment by Birchedahl** [comment]
ja

 

Med venlig hilsen

Anne Birchedahl Elfengård

Åtoften 28

2680 Solrød str. 

2390 0028

 

Fra: Christian Reinholdt ***@***.***> 
Sendt: 12. juni 2023 18:01
Til: rndfm/elspotpris ***@***.***>
Cc: Birchedahl ***@***.***>; Author ***@***.***>
Emne: Re: [rndfm/elspotpris] Elafgifter mangler i beregningerne (Discussion #173)

 

Begge er valgt med samme netselskab? 

*Med venlig hilsen | best regards* 
*Christian Reinholdt* 
Web-professionel og autodidakt alt-mulig-mand. 

tlf. 31718115 
***@***.*** <mailto:***@***.***>
